### PR TITLE
docs: refactor several README files

### PR DESCRIPTION
**Pull request by wingkwong**
## Change Summary

- Added ``COMMIT_CONVENTION.md`` and ``CONTRIBUTING.md`` under ``.github`` directory.
- Updated ``README.md`` at root directory
  - Added conventional commits badge 
  - Added Table of Contents
  - Updated Contributing link
  - Removed duplicate info 
 - Updated contribution guide link in ``noco-docs``
 - Removed the old contributing directory. The content has been merged to ``.github/CONTRIBUTING.md``

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [x] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [x] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Additional information

README files under ``nocodb/scripts/markdown/readme/languages`` will be updated accordingly after this PR is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Turns any MySQL, PostgreSQL, SQL Server, SQLite & MariaDB into a smart-spreadshe
  
 [![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)](https://travis-ci.com/github/NocoDB/NocoDB) 
 [![Node version](https://badgen.net/npm/node/next)](http://nodejs.org/download/)
-[![Twitter](https://img.shields.io/twitter/url/https/twitter.com/NocoDB.svg?style=social&label=Follow%20%40NocoDB)](https://twitter.com/NocoDB)
```
Comment (o1lab) [inline README.md:17]: Let us retain the twitter badge ?

File: README.md [hunk 2]
```diff
@@ -88,13 +128,6 @@ npm start
 ### GUI
 Access Dashboard using : [http://localhost:8080/dashboard](http://localhost:8080/dashboard)
 
-
-# Join Our Community
-<a href="https://discord.gg/5RgZmkW">
-<img src="https://discordapp.com/api/guilds/661905455894888490/widget.png?style=banner3" alt="">
-</a>
-<br>
-
```
Comment (o1lab) [inline README.md:97]: Let us retain discord badge

File: packages/noco-docs/content/en/index.md [hunk 3]
```diff
@@ -89,7 +85,7 @@ We use ``Lerna`` to manage multi-packages. We have the following [packages](http
 
 ## Contributions
 
-All contributions are welcome. NocoDB projects can be found in the [NocoDB](https://github.com/nocodb) Github organization. Our core uses [The AGPL V3 license](https://github.com/nocodb/nocodb/blob/master/LICENSE) and all contributors should read the [contribution guides](https://github.com/nocodb/nocodb/tree/master/scripts/contribute) before making your first contribution. 
+All contributions are welcome. NocoDB projects can be found in the [NocoDB](https://github.com/nocodb) Github organization. Our core uses [The AGPL V3 license](https://github.com/nocodb/nocodb/blob/master/LICENSE) and all contributors should read the [contribution guides](https://github.com/nocodb/nocodb/blob/master/.github/CONTRIBUTING.md) before making your first contribution.
```
Comment (o1lab) [inline packages/noco-docs/content/en/index.md:88]: Cool, please confirm if we are not referring to ./contribute folder from docs as such too.

File: README.md [hunk 4]
```diff
@@ -46,9 +50,40 @@ Turns any MySQL, PostgreSQL, SQL Server, SQLite & MariaDB into a smart-spreadshe
...
+  * [Running Backend locally](#running-backend-locally)
+  * [Running Frontend locally](#running-frontend-locally)
+  * [Running Cypress tests locally](#running-cypress-tests-locally)
+- [Contributing](#contributing)
+- [Why are we building this?](#---why-are-we-building-this-)
+- [Our Mission](#--our-mission--)
+- [Contributors](#contributors-------------)
+
+
+
```
Comment (o1lab) [inline README.md:85]: Put it after the images

File: README.md (hunk 2 above)
Comment (wingkwong) [reply README.md:97]: added back

**Pull request merged**